Add missing properties (#1626)
redpanda-data/documentation: docs/reference/tunable-properties.mdx

---

### cloud_storage_credentials_host

> Hostname to connect to for retrieving role based credentials. If `null`, hostname is derived from [`cloud_storage_credentials_source`](#cloud_storage_credentials_source). Only required when using IAM role based access.

Reviewer: ```suggestion
Hostname to connect to for retrieving role-based credentials. If `null`, hostname is derived from [`cloud_storage_credentials_source`](#cloud_storage_credentials_source). Only required when using IAM role based access.
```

**Type**: string

**Default**: null

**Restart required**: yes

---

### cloud_storage_credentials_source

> The source of credentials to connect to cloud services. If [`cloud_storage_credentials_host`](#cloud_storage_credentials_host) is `null`, this is used to derive the hostname to connect to retrieve role based credentials.

Reviewer: ```suggestion
The source of credentials to connect to cloud services. If [`cloud_storage_credentials_host`](#cloud_storage_credentials_host) is `null`, this is used to derive the hostname to connect to retrieve role-based credentials.
```

---

### cloud_storage_graceful_transfer_timeout_ms

> Maximum duration to wait for uploads to complete before a leadership transfer. If `null`, leadership transfers proceed without waiting.

Reviewer: ```suggestion
Maximum duration to wait for uploads to complete before a leadership transfer. If `null`, leadership transfers proceed with no delay.
```
